FirebaseAdmin/FirebaseAdmin/Auth/UserRecord.cs

```diff
@@ -120,6 +123,21 @@ internal UserRecord(GetAccountInfoResponse.User user)
         /// </summary>
         public IUserInfo[] ProviderData { get; }
 
+        /// <summary>
```

Comment:
Sorry if I wasn't clear in my earlier review. But these properties are not needed. CreatedAt and LastLoginAt are already in UserMetadata and ValidSince is same as TokensValidAfterTimestamp